src/hotspot/cpu/riscv/globals_riscv.hpp

```diff
@@ -101,6 +101,7 @@ define_pd_global(intx, InlineSmallCode,          1000);
   product(bool, UseRVA20U64, true, "Use RVA20U64 profile")                       \
   product(bool, UseRVC, false, "Use RVC instructions")                           \
   product(bool, UseRVA22U64, false, EXPERIMENTAL, "Use RVA22U64 profile")        \
+  product(bool, UseRVA23U64, false, EXPERIMENTAL, "Use RVA23U64 profile")        \
```

Comment:
Can we group the three RV profile-related options together?

Reply:
Sure, will do